content/healthchecks/deploy.files/liveness-app.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
Based on other suggestion in liveness.md, move this file to `content/healthchecks/liveness.files/liveness-app.yaml`